product/components.mdx

```diff
@@ -1,12 +1,39 @@
 ---
-title: "Components"
-description: "Makeswift leverages the power of React to give you unparalleled flexibility and control over your website's design and functionality on an ongoing basis."
+title: "Components and Elements"
 icon: "cube" iconType: "solid"
 ---
 
+Makeswift leverages the power of React to give you unparalleled flexibility and control over your website's design and functionality on an ongoing basis.
```

Comment:
Would it make sense to not mention react to start since for visually created components, code doesn't factor in?